work10/index.js

```diff
@@ -28,9 +28,17 @@ function getData() {
 async function displayView() {
   loading();
 
-  const arrayData = await getData();
-  createElement(arrayData);
-  hideLoading();
+  let arrayData;
```

Comment:
【IMO】
Why did you use ”let” for this variable?
I thought that if it was ”let”, the data could be overwritten.

Reply:
I think the same way.

Reply:
The reason I used "let" for the variable was because when I wrote the code using "const" I got an error that it couldn't be reassigned.
The cause was constarray Data ;.
As a feature of const, I didn't understand that variable declarations alone would cause an error.
example:
const k;
/ const declaration has no initializer

02f6423